playbooks/upgrade-ha-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Install and config openlabcmd
+  hosts: nodepool-master, nodepool-slave
+  become: yes
+  tasks:
+    - name: upgrade nodepool
+      shell: |
```

Comment:
Should we consider to set the `set -e` to enable the error exit to do short exit when error happen.